pagopa/io-app
# fix: [IOBP-53] Fix IDPay initiative bonus counters

Depends on https://github.com/pagopa/io-app/pull/4639

## Short description
This PR fixes the correct display of the available and refundable bonus amounts in initiative screen.

## List of changes proposed in this pull request
- `InitiativeDetailsScreen.tsx`: added correct amounts to display

## How to test
With the `io-dev-server`, navigate to the detail screen of an IDPay initiative and check if there are correct amountsin bonus counters
